templates/config.edn

```diff
@@ -343,7 +343,12 @@
  ;;     ;use Percent-encoding for other invalid characters
  :file/name-format :triple-lowbar
 
- ;; specify the format of the filename for journal files
+ ;; Specify the format of the filename for journal files.
+ ;; Warning:
+ ;;   Changing this setting may cause issues if not configured correctly.
+ ;;   After changing this setting, you must reindex.
+ ;; Available values:
+ ;;  - "yyyy-MM-dd" (default)
```

Comment:
How to apply this change to all existing files? Will re-indexing convert all yyyy_MM_dd to yyyy-MM-dd?

Reply:
No, it will not convert existing files. So use it with caution.

Reply:
I should add a note about it not being retroactive.

Reply:
@cnrpman is there a need to request users to re-index? I think it would work without it starting the next day but I didn't test it recently.

Reply:
re-index is not required I think